### ISSUE TITLE
How do I install this?

### ISSUE THREAD

**Issue by audunmo** [spyder-ide/spyder-vim#33]
The current installation instructions don't work. Running `python setup.py install` produces a value error:

`ValueError: bad marshal data (unknown type code)`

**Comment by andfoy** [comment]
@audun- We haven't released Python packages yet. However, if you're installing from source, you can execute ``pip install .`` on the repo's root.

**Comment by VeloSteve** [comment]
I'm a little confused here.  Are the commands
pip install spyder-vim
and
conda install spyder-vim -c spyder-ide
as shown in README.md not supposed to work?

I understand that if it doesn't work there may be no resources to fix the problem, but it seems
that deleting those lines from the README wouldn't take long, and would prevent a lot of wasted time.

**Comment by andfoy** [comment]
@VeloSteve Thanks for your recommendation, right now we're pending of doing the first release. There's already an open PR (#34) to solve this issue, feel free to add your contribution.

**Comment by collinstarkweather** [comment]
There seems to have been no movement on this for over two years and the pip and conda install procedures do not work.  

I tried manually installing it in an Anaconda environment on Windows per [PR #34](https://github.com/spyder-ide/spyder-vim/pull/34) with the following:  

```
git clone https://github.com/spyder-ide/spyder-vim
cd spyder-vim
python setup.py install .
```
However, it broke things so badly that neither uninstalling and reinstalling Spyder nor the entire Anaconda distribution fixed it. 

Basically, the install (at least in my case on Windows) seems to be a complete disaster.

**Comment by ccordoba12** [comment]
We'll create a proper release for this package once @bgallois let us know that he's happy with the current state of things.

**Comment by mochsner** [comment]
@ccordoba12  @bgallois  Are ya'll still working on this? I'm on the newest version of spyder and getting the following on Windows.
```
!pip install spyder-vim
---RUNNING NEW PROGRAM---
C:\Users\Marc Ochsner\AppData\Local\Programs\Spyder\Python\python.exe
0.12.0
ERROR: Could not find a version that satisfies the requirement spyder-vim
ERROR: No matching distribution found for spyder-vim
```

**Comment by AlexanderBlum** [comment]
I have the same question - was stoked to install this, then disappointed when I got the same error shown by mochsner...

**Comment by sdsaati** [comment]
I have the same problem too

**Comment by dalthviz** [comment]
Closing as duplicate of #21 

To follow up the progress towards the initial release for this plugin/package you can check https://github.com/spyder-ide/spyder-vim/issues/78
